specification/apicenter/ApiCenter.Management/models.tsp

```diff
@@ -266,6 +267,37 @@ model MetadataSchemaProperties {
   assignedTo?: MetadataAssignment[];
 }
 
+@added(Versions.v2024_06_01_preview)
+@doc("The import from Azure API Management request.")
+model ImportFromApimRequest {
+  @doc("An entity the metadata schema is requested for.")
+  sourceResourceIds?: string[];
+}
+
+@added(Versions.v2024_06_01_preview)
+@doc("The import operation was successfully completed.")
+model ImportFromApimSuccessResult {
+  /**
+   * The successful response status code.
+   */
+  @statusCode code: 200;
+
+  /**
+   * Async operation status.
+   */
+  status: string;
```

Comment:
>string [](http://example.com/codeflow?start=10&length=6)

are these from set a values that this can be enums ?

Reply:
Removed this functionality temporarily, will go to next version.